### ISSUE TITLE
Error with nativescript-contact after migrating to nativescript 8

### ISSUE THREAD

**Issue by Bidou44** [NativeScript/NativeScript#9345]
**Environment**
✔ Component nativescript has 8.0.1 version and is up to date.
✔ Component @nativescript/core has 8.0.1 version and is up to date.
✔ Component @nativescript/ios has 8.0.0 version and is up to date.
✔ Component @nativescript/android has 8.0.0 version and is up to date.

**Describe the bug**
After migrating my application from nativescript 7 to nativescript 8, I got the following error:

ERROR in ./node_modules/nativescript-contacts/contact-helper.android.js 1:16-38
Module not found: Error: Can't resolve 'application' in '/Users/xxx/node_modules/nativescript-contacts'
 @ ./node_modules/nativescript-contacts/index.android.js 5:13-40
 ...

ERROR in ./node_modules/nativescript-contacts/contact-helper.android.js 3:18-41
Module not found: Error: Can't resolve 'image-source' in '/Users/xxx/node_modules/nativescript-contacts'
 @ ./node_modules/nativescript-contacts/index.android.js 5:13-40
 ...

ERROR in ./node_modules/nativescript-contacts/contact-model.android.js 2:16-38
Module not found: Error: Can't resolve 'application' in '/Users/xxx/node_modules/nativescript-contacts'
 @ ./node_modules/nativescript-contacts/index.android.js 2:14-40
 ...

ERROR in ./node_modules/nativescript-contacts/contact-model.android.js 4:18-41
Module not found: Error: Can't resolve 'image-source' in '/Users/xxx/node_modules/nativescript-contacts'
 @ ./node_modules/nativescript-contacts/index.android.js 2:14-40
 ...

ERROR in ./node_modules/nativescript-contacts/group-model.android.js 3:16-38
Module not found: Error: Can't resolve 'application' in '/Users/xxx/node_modules/nativescript-contacts'
 @ ./node_modules/nativescript-contacts/index.android.js 4:12-36
 ...

ERROR in ./node_modules/nativescript-contacts/index.android.js 1:16-38
Module not found: Error: Can't resolve 'application' in '/Users/xxx/node_modules/nativescript-contacts'
 ...

**To Reproduce**
ns create myCoolApp
add package     "nativescript-contacts": "~1.6.2"
open the contact popup

      getContact().then((result: GetContactResult) => {
          if (result.response === 'selected') {
             
          }
      }).catch(e => {
          console.log(e);
      });

ns run android

**Expected behavior**
The contact plugin should work as in nativescript 7

**Sample project**

**Additional context**
angular 11.2.10

**Comment by triniwiz** [comment]
This plugin uses the old short imports which isn’t supported, it is possible to add some fallbacks to map them to the new module name we recommend using the full package name

**Comment by Bidou44** [comment]
Thanks, but how can I add those fallbacks to remap them? is it in webpack?
Since I'm not the owner of the nativescript-contacts plugin I cannot change the code directly. ...

Any idea?

**Comment by justintoth** [comment]
@Bidou44 Did you figure out how to fix this error? I'm running into the same with the nativescript-purchase plugin. Or @triniwiz can you explain how to add the fallbacks?

**Comment by lucianomurr** [comment]
Hi All!
Any update on this?

**Comment by sir0pip** [comment]
I solved the problem I was having with nativescript-contacts with a change in the contact-model.android.js and contact-model.ios.js files for the plugin.

Remove "/image-source" from:
var imageSource = require('@nativescript/core/image-source');
=>
var imageSource = require('@nativescript/core');

This is a stop-gap though, as any refresh of the plugin or loading the plugin on a new instance of the application will revert the change back.  I've submitted a pull request with this change to Firescript.
